packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -368,13 +368,16 @@ async function handler(req: CustomRequest) {
   await Promise.all(promises);
 
   const workflows = await getAllWorkflowsFromEventType(bookingToDelete.eventType, bookingToDelete.userId);
+  const parsedMetadata = bookingMetadataSchema.safeParse(bookingToDelete.metadata || {});
```

Comment:
Will `parsedMetadata.success` be false if `bookingToDelete.metadata || {}` result in `{}`?

Reply:
Nah, it would be true .`videoCallUrl` is an optional string